.ci/azure/linux_intel_gna.yml

```diff
@@ -0,0 +1,227 @@
+    submodules: 'true'
+    path: openvino
+
+  - checkout: openvino_contrib
+    clean: 'true'
+    fetchDepth: '1'
+    submodules: 'true'
+    path: openvino_contrib
+
+  - script: |
+      set -e
+      sudo -E $(REPO_DIR)/install_build_dependencies.sh
+      # Move jdk into contrib
+      # 'clang' compiler is to check that samples can be built using it
+      sudo apt --assume-yes install openjdk-11-jdk libbz2-dev clang
+      # Speed up build
+      sudo apt -y --no-install-recommends install unzip
+      wget https://github.com/ninja-build/ninja/releases/download/v1.10.2/ninja-linux.zip
+      unzip ninja-linux.zip
+      sudo cp -v ninja /usr/local/bin/
+    displayName: 'Install dependencies'
+
+    # Should be after 'Install dependencies' because Git lfs is not installed
+  - checkout: testdata
+    clean: 'true'
+    fetchDepth: '1'
+    lfs: 'true'
+    path: testdata
+
+  - task: CMake@1
+    inputs:
+      # CMake must get Python 3.x version by default
+      cmakeArgs: >
+        -GNinja
+        -DCMAKE_VERBOSE_MAKEFILE=ON
+        -DCMAKE_BUILD_TYPE=$(BUILD_TYPE)
+        -DENABLE_TESTS=ON
+        -DENABLE_STRICT_DEPENDENCIES=OFF
+        -DOPENVINO_EXTRA_MODULES=$(OPENVINO_CONTRIB_REPO_DIR)/modules
+        -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
+        -DCMAKE_C_COMPILER_LAUNCHER=ccache
+        -DENABLE_INTEL_GNA=ON
+        -DENABLE_INTEL_GNA_DEBUG=ON
+        -DENABLE_INTEL_CPU=OFF
+        -DENABLE_INTEL_GPU=OFF
+        -DENABLE_OV_TF_FRONTEND=OFF
+        -DENABLE_OV_PADDLE_FRONTEND=OFF
+        -DENABLE_OV_PYTORCH_FRONTEND=OFF
+        -DENABLE_OV_ONNX_FRONTEND=OFF
+        -DENABLE_PYTHON=OFF
+        -DBUILD_nvidia_plugin=OFF
+        -DENABLE_PYTHON=OFF
+        -S $(REPO_DIR)
+        -B $(BUILD_DIR)
+    displayName: 'Cmake OpenVINO'
+
+  - script: ls -alR $(REPO_DIR)/temp/
+    displayName: 'List temp SDKs'
+
+  - script: ccache --zero-stats --max-size=50G --show-config
+    displayName: 'Clean ccache stats'
+
+  - script: cmake --build $(BUILD_DIR) --parallel --config $(BUILD_TYPE) --target openvino_intel_gna_plugin
+    env:
+      CCACHE_DIR: $(CCACHE_DIR)
+      CCACHE_TEMPDIR: $(TMP_DIR)/ccache
+      CCACHE_BASEDIR: $(Pipeline.Workspace)
+      CCACHE_MAXSIZE: 50G
+    displayName: 'Build Lin Intel GNA'
+
+  - script: ccache --show-stats
+    displayName: 'Show ccache stats'
+
+  - script: ls -alR $(REPO_DIR)/bin/
+    displayName: 'List bin files'
+
+  - script: sudo apt-get remove libtbb2 -y
+    displayName: 'Remove debian dependencies'
+    condition: eq(variables['CMAKE_CPACK_GENERATOR'], 'DEB')
+
+  - script: |
+      set -e
+      cmake -DCMAKE_INSTALL_PREFIX=$(INSTALL_DIR) -DCOMPONENT=tests -P $(BUILD_DIR)/cmake_install.cmake
+    displayName: 'Install tests'
+
+  - script: ls -alR $(INSTALL_DIR)
+    displayName: 'List install test files'
+
+  - script: $(RUN_PREFIX) $(INSTALL_TEST_DIR)/ov_gna_unit_tests --gtest_output=xml:$(INSTALL_TEST_DIR)/TEST-ov_gna_unit_tests.xml
+    displayName: 'GNA UT'
```

Comment:
how is It different from https://github.com/openvinotoolkit/openvino/blob/master/.ci/azure/linux.yml#L432 ?